🐞 Bug version.rb cannot find ruby_libversion

```
$ ./version.rb 
<internal:/usr/local/lib64/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require': cannot load such file -- ruby_libversion (LoadError)
Did you mean?  rubygems/version
	from <internal:/usr/local/lib64/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:136:in `require'
	from ./version.rb:12:in `<main>'
```
Related?
```
$ gem install ruby-libversion
Building native extensions. This could take a while...
ERROR:  Error installing ruby-libversion:
	ERROR: Failed to build gem native extension.

    current directory: /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0/ext/libversion
/usr/local/bin/ruby extconf.rb
checking for pkg-config for libversion... not found
creating Makefile

current directory: /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0/ext/libversion
make DESTDIR\= sitearchdir\=./.gem.20240610-858-dp0xke sitelibdir\=./.gem.20240610-858-dp0xke clean

current directory: /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0/ext/libversion
make DESTDIR\= sitearchdir\=./.gem.20240610-858-dp0xke sitelibdir\=./.gem.20240610-858-dp0xke
compiling libversion.c
libversion.c:2:10: fatal error: libversion/version.h: No such file or directory
    2 | #include "libversion/version.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:248: libversion.o] Error 1

make failed, exit code 2

Gem files will remain installed in /usr/local/lib64/ruby/gems/3.3.0/gems/ruby-libversion-1.0.0 for inspection.
Results logged to /usr/local/lib64/ruby/gems/3.3.0/extensions/x86_64-linux-gnu/3.3.0/ruby-libversion-1.0.0/gem_make.out
```

--- a/manifest/armv7l/l/libversion.filelist
+++ b/manifest/armv7l/l/libversion.filelist
@@ -1,0 +1,12 @@
+/usr/local/bin/version_compare
+/usr/local/bin/version_sort
+/usr/local/include/libversion/config.h
+/usr/local/include/libversion/export.h
+/usr/local/include/libversion/version.h
+/usr/local/lib/cmake/libversion/libversionConfig-release.cmake
+/usr/local/lib/cmake/libversion/libversionConfig.cmake
+/usr/local/lib/libversion.a
+/usr/local/lib/libversion.so
+/usr/local/lib/libversion.so.1
+/usr/local/lib/libversion.so.1.0.0
+/usr/local/lib/pkgconfig/libversion.pc

--- a/manifest/i686/l/libversion.filelist
+++ b/manifest/i686/l/libversion.filelist
@@ -1,0 +1,12 @@
+/usr/local/bin/version_compare
+/usr/local/bin/version_sort
+/usr/local/include/libversion/config.h
+/usr/local/include/libversion/export.h
+/usr/local/include/libversion/version.h
+/usr/local/lib/cmake/libversion/libversionConfig-release.cmake
+/usr/local/lib/cmake/libversion/libversionConfig.cmake
+/usr/local/lib/libversion.a
+/usr/local/lib/libversion.so
+/usr/local/lib/libversion.so.1
+/usr/local/lib/libversion.so.1.0.0
+/usr/local/lib/pkgconfig/libversion.pc

--- a/manifest/x86_64/l/libversion.filelist
+++ b/manifest/x86_64/l/libversion.filelist
@@ -1,0 +1,12 @@
+/usr/local/bin/version_compare
+/usr/local/bin/version_sort
+/usr/local/include/libversion/config.h
+/usr/local/include/libversion/export.h
+/usr/local/include/libversion/version.h
+/usr/local/lib64/cmake/libversion/libversionConfig-release.cmake
+/usr/local/lib64/cmake/libversion/libversionConfig.cmake
+/usr/local/lib64/libversion.a
+/usr/local/lib64/libversion.so
+/usr/local/lib64/libversion.so.1
+/usr/local/lib64/libversion.so.1.0.0
+/usr/local/lib64/pkgconfig/libversion.pc

--- a/packages/core.rb
+++ b/packages/core.rb
@@ -3,7 +3,7 @@ require 'package'
 class Core < Package
   description 'Core Chromebrew Packages.'
   homepage 'https://github.com/chromebrew/chromebrew'
-  version '2.3'
+  version '2.4'
   license 'GPL-3+'
   compatibility 'all'
 
@@ -52,6 +52,7 @@ class Core < Package
   depends_on 'libtirpc'
   depends_on 'libunbound'
   depends_on 'libunistring'
+  depends_on 'libversion'
   depends_on 'libxml2'
   depends_on 'libyaml'
   depends_on 'lz4'

--- a/packages/libversion.rb
+++ b/packages/libversion.rb
@@ -1,0 +1,21 @@
+require 'buildsystems/cmake'
+
+class Libversion < CMake
+  description 'Advanced version string comparison library.'
+  homepage 'https://github.com/repology/libversion'
+  version '3.0.3'
+  license 'MIT'
+  compatibility 'all'
+  source_url 'https://github.com/repology/libversion.git'
+  git_hashtag version
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: '3fb7fd997c1de9d3d567b80931b2bea977ad233d55f8a067bc93bbbd8f7b8e85',
+     armv7l: '3fb7fd997c1de9d3d567b80931b2bea977ad233d55f8a067bc93bbbd8f7b8e85',
+       i686: '63a3c352fa9a6c48199581dcaaa681efd323ad2fc5d643237ee34f7057acc819',
+     x86_64: '28f8483402a4b9c465bd07fb2907c386ef2227441fbcd5fe505d0618fc42f626'
+  })
+
+  run_tests
+end

